shared-data/python/Makefile

```
@@ -131,3 +131,8 @@ deploy: wheel
 .PHONY: test
 test:
 	$(python) -m pytest --cov=opentrons_shared_data --cov-report xml:coverage.xml tests
+
+.PHONY: command_schema
+command_schema: 
+	$(python) opentrons_shared_data/command/generate_command_schema.py $(version) > ../command/schemas/$(version).json
```

```suggestion
.PHONY: command-schema
command-schema: 
	$(python) opentrons_shared_data/command/generate_command_schema.py $(COMMAND_SCHEMA_VERSION) > ../command/schemas/$(COMMAND_SCHEMA_VERSION).json
```

I think [makefile convention](https://stackoverflow.com/a/62042656/497934) is for targets to be `kebab-case`, and for variables supplied from the outside to be `UPPERCASE`.

Expanding `version` to `COMMAND_SCHEMA_VERSION` is to avoid collisions with anything else. Probably unnecessary, but I guess I'm nervous because makefile variables are basically unscoped globals.